fala/templates/generic-py-base.html

```diff
@@ -75,38 +79,31 @@
         document.cookie = key + '=' + value + ';expires=' + expires.toUTCString();
       }
 
-      function getCookie(key) {
-        var keyValue = document.cookie.match('(^|;) ?' + key + '=([^;]*)(;|$)');
-        return keyValue ? keyValue[2] : null;
-      }
-
-      function eraseCookie(key) {
-        var keyValue = getCookie(key);
-        setCookie(key, keyValue, '-1');
-      }
-
       $(function() {
         $("#cookieAccept").click(function() {
           setCookie("cookiePermission", "Allowed", 30);
           $(".laa-cookie-banner").addClass("laa-cookies-accepted");
           tagMan();
         });
         $("#cookieReject").click(function() {
-          eraseCookie("cookiePermission");
+          setCookie("cookiePermission", "Rejected", 30);
           $(".laa-cookie-banner").addClass("laa-cookies-rejected");
         });
-        $("#cookieReview").click(function() {
-          $(".laa-cookie-banner").removeClass("laa-cookies-accepted").removeClass("laa-cookies-rejected");
-          $("html, body").animate(
-            { scrollTop: 0 },
-            "slow",
-            "swing",
-            function(){
-              $("#cookieAccept").focus();
-            }
-          );
-          return false;
-        });
+      });
+      $("#cookie_policy_page_choice").submit(function(e){
+        e.preventDefault();
+        var value = $("input[name='cookies']:checked").val();
```

Comment:
Again this fine as it's being replaced. But make sure in future to use `let` and not `var` here.